rosdep/base.yaml

```diff
@@ -819,7 +819,7 @@ gfortran:
   arch: [gcc-fortran]
   debian: [gfortran]
   fedora: [gcc-fortran]
-  gentoo: [sys-devel/gcc]
+  gentoo: ['sys-devel/gcc[fortran]']
```

Comment:
So, for a user that doesn't do `*/* -*`, if they use one of the desktop profiles, this flag would not be enabled?